gnosis/safe-react
# Fix: AddressInput freezing the UI when using a QR code

## What it solves
We noticed that e2e tests (namely, Load Safe) started failing.

The reason was the Address Input form was freezing after inserting a value from a QR code. There was an infinite recursion.

## How this PR fixes it
I reverted #2930 and simplified the ENS/UD resolving indicator.

## How to test it
* Go to Load Safe
* Load a QR code in the address input
* It should not freeze the UI
<img width="321" alt="Screenshot 2021-11-10 at 17 18 00" src="https://user-images.githubusercontent.com/381895/141160716-2572dbf2-d9e2-4412-9007-24ffc3b54328.png">
